From 6834566f9c5ce1a1988763f5faf0cb6686d73f29 Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Fri, 14 Feb 2020 09:34:22 -0600
Subject: [PATCH 036/120] dt-bindings, firmware: add compatible value Intel
 Stratix10 service layer binding

commit 7951efcfb403c9d062a1edc298989127acf10f53 from
https://github.com/altera-opensource/linux-socfpga.git

[upstream commit e99ede177d0ac6858d937fbe56f0560aa7fb8b82]

A a compatible property value to Intel Stratix10 service layer binding

Signed-off-by: Richard Gong <richard.gong@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 .../devicetree/bindings/firmware/intel,stratix10-svc.txt        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/firmware/intel,stratix10-svc.txt b/Documentation/devicetree/bindings/firmware/intel,stratix10-svc.txt
index 1fa66065acc6..6eff1afd8daf 100644
--- a/Documentation/devicetree/bindings/firmware/intel,stratix10-svc.txt
+++ b/Documentation/devicetree/bindings/firmware/intel,stratix10-svc.txt
@@ -23,7 +23,7 @@ Required properties:
 The svc node has the following mandatory properties, must be located under
 the firmware node.
 
-- compatible: "intel,stratix10-svc"
+- compatible: "intel,stratix10-svc" or "intel,agilex-svc"
 - method: smc or hvc
         smc - Secure Monitor Call
         hvc - Hypervisor Call
-- 
2.31.1

